src/org/traccar/database/AttributesManager.java

```diff
@@ -26,9 +26,14 @@ public AttributesManager(DataManager dataManager) {
     }
 
     @Override
+    public Attribute getById(long calendarId) {
```

Comment:
It is mistake here, variable should be named `attributeId` of course.

I also thought about it, but I'm not familiar with it, It will take time.